javascript/calcs/equip/objects/equip_buffs_list.js

```diff
@@ -389,6 +391,8 @@ equip_buff_list = [
 
     //code essencia
     { "id": 110, "name": "Electron Upgrade", "category": "class", "class": ["code_essencia"], "icon": "electron_upgrade", "mode": "mult", "options": ["Basic"], "stats": ["as", "ms"], "values": [[10, 10]] },
+    { "id": 691, "name": "Queen's Authority", "category": "class", "class": ["code_essencia"], "icon": "queens_authority", "mode": ["mult", "mult_red"], "options": ["10 stacks", "9 stacks", "8 stacks", "7 stacks", "6 stacks", "5 stacks", "4 stacks", "3 stacks", "2 stacks", "1 stack"], "stats": ["specialdmg_mod", "cdr_mod"], "values": [[20, 20, 20, 20, 20, 20, 20, 20, 20, 20], [20, 18, 16, 14, 12, 10, 8, 6, 4, 2]], "pvp_values": [[2, 2, 2, 2, 2, 2, 2, 2, 2, 2], [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]] },
```

Comment:
the lists should be inverted (the top level list is options, bottom level list is stats)